schemas/layer-3.cue

```diff
@@ -0,0 +1,70 @@
+// layer-3-policy.cue
+package layer3
+
+import "time"
+
+// TODO
+// Create schema for a policy document
+// Policy docs should reference common and unique risk definitions, so we need schema for that as well
+
+#Policy: {
+    // Metadata useful for evaluation and automation
+    metadata?: {
+        // version is the version of the policy itself
+        version: string @go(Version)
+        // owner is the organizational unit responsible for the policy. This could be a department, team, or person.
+        owner: string @go(Owner)
+        // last_modified is the date and time when the policy was last modified
+        last_modified: time.Time @go(LastModified)
+        // the version of the Simplified Compliance Infrastructure model/schema used to create this policy
+        sci_version: string @go(SCIVersion)
+        // useful commentary or notes about the policy
+        remarks: string @go(Remarks)
+    }
+
+    // Unique identifier for this policy
+    id: string @go(ID)
+
+    // Human-readable title of the policy
+    title: string @go(Title)
+
+    // Short description of this policy’s intent or purpose
+    description: string @go(Description)
+
+    // Optional reference to a parent policy this one inherits from or refines
+    parent_policy_id?: string @go(ParentPolicyID)
+
+    // Policy classification level (e.g., mandatory, recommended)
+    classification: "mandatory" | "recommended"  @go(Classification)
+
+    // Reference to one or more Layer 2 control catalogs
+    control_catalogs: [#CatalogReference, ...#CatalogReference] @go(ControlCatalogs)
+
+}
+
+#CatalogReference: {
+    // Unique identifier for the catalog
+    id: string @go(ID)
+    // version of the catalog
+    version: string @go(Version)
+
+    // List of IDs of Layer 2 Catalog.Metadata.ApplicabilityCategories
+    applicability: [...string] @Go(ApplicabilityCategories)
+    // one or more modifications to the controls in this catalog
+    modifications?: [#ControlModification, ...#ControlModification] @go(Modifications)
+
+    // Reason for including this catalog in the policy
+    objective?: string @go(Objective)
+}
+
+#ControlModification: {
```

Comment:
It would likely require an update to Layer 2, but I think organizational defined parameters could be useful as part of the control modification. Layer 2 could define possible parameters with associated constraints, perhaps even customizing by level.